java/com.oracle.mxtool.compilerserver/src/com/oracle/mxtool/compilerserver/CompilerDaemon.java

```diff
@@ -134,11 +145,6 @@ public void run() {
                         logf("Result = %d\n", result);
 
                         output.write(result + "\n");
-
-                        // Try close open file handles (may help on Windows)
-                        System.gc();
-                        System.gc();
-                        System.runFinalization();
```

Comment:
These statements should never have been added in the first place. The problem they were aimed at has a completely different cause (see #99) and they have been observed to cause the daemon to slow down significantly on Linux.